store/mockstore/mocktikv/cluster.go

```diff
@@ -665,3 +672,25 @@ func newStore(storeID uint64, addr string, labels ...*metapb.StoreLabel) *Store
 		},
 	}
 }
+
+func (s *Store) mergeLabels(labels []*metapb.StoreLabel) {
+	if len(s.meta.Labels) < 0 {
```

Comment:
`len` should never return a value `< 0` :smile:

Reply:
updated.